src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -40,13 +40,19 @@
 long-summary: >
     Update the properties of a storage account's blob service, including
     properties for Storage Analytics and CORS (Cross-Origin Resource
-    Sharing) rules. But currently we only support enabling or disabling change feed for a storage account's blob service.
+    Sharing) rules. But currently we only support enabling or disabling 'change feed' and 'delete retention policy' for a storage account's blob service.
 parameters:
   - name: --enable-change-feed
     short-summary: 'Indicate whether change feed event logging is enabled. If it is true, you enable the storage account to begin capturing changes. The default value is true. You can see more details in https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-change-feed?tabs=azure-portal#register-by-using-azure-cli'
+  - name: --delete-retention
+    short-summary: 'Indicate whether delete retention policy is enabled for the blob service.'
+  - name: --delete-retention-period
```

Comment:
I think --delete-retention-days may be better.

Reply:
OK, I've changed it